server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java

```diff
@@ -284,7 +284,7 @@ public Coordinator(
         this.nodeHealthService = nodeHealthService;
     }
 
-    private ClusterFormationState getClusterFormationState() {
+    public ClusterFormationState getClusterFormationState() {
```

Comment:
@DaveCTurner does it make sense to send this over if the discovery cluster formation timeout has not lapsed yet?